tools/src/test/scala/scala/scalanative/LinkerSpec.scala

```diff
@@ -22,7 +21,7 @@ abstract class LinkerSpec extends AnyFlatSpec {
    * @param sources Map from file name to file content representing all the code
    *                to compile and link.
    * @param driver  Optional custom driver that defines the pipeline.
-   * @param fn      A function to apply to the products of the compilation.
+   * @param f      A function to apply to the products of the compilation.
```

Comment:
Is this change intended?   Seems like a change with no distinction.
If intended, I believe this line needs an additional blank to line up "A function"
with those above.  It also needs a change from f to fn in the line below.

In many other places in the world fn is used as shorthand for function.
Does changing to f add value here? especially when there are no explicit 
parameters?

Agreed, that in the overall scheme of things this is in the atto-cosm.
It just jumped out at me as I was trying to understand the changes which are the
intent of this PR.

Reply:
Looks like this `fix` was added by mistake, I'll remove this. Only thing to which it would give some additional value might be using scaladoc - I'm not sure about this one, but I think scaladoc might throw some errors as there is no `fn` param, but `f` instead. Anyway such changes (targeting scaladoc) should go to seperate PR, e.g fixing #1121